tests/test_database.py

```
@@ -42,6 +42,17 @@ def test_init_db_is_idempotent(db_session):
     assert table_names_first_call == table_names_second_call
 
 
+def test_has_schema_objects_false_for_empty_db(db_session):
```

Ruff (0.15.9)

[warning] Unused function argument: `db_session`

(ARG001)